### ISSUE TITLE
Noticeable lag while starting and stopping Ktor

### ISSUE THREAD

**Issue by AdityaAnand1** [ktorio/ktor#1451]
I've noticed that (even in hello world projects)
- After starting the server, it takes 4-5 seconds for it to start accepting API calls on localhost
- Upon stopping the server, the process often lingers around for another 5-6 seconds showing the below icon 👇 

![image](https://user-images.githubusercontent.com/13274079/69169052-b52ffe80-0b1d-11ea-82e2-6a41d38a601f.png)

Now this may not sound like much, but starting and stopping every day hundreds of times, this could easily get very annoying.

So I wonder, if I'm missing something or whether this is something that everyone faces? Is there a way to fix this?

**Comment by Koriit** [comment]
I think it is very likely caused by the engine underneath. Which one are you using? Personally, I have experienced something similar with Netty and Jetty engines. CIO seems to start/stop almost instantly.

**Comment by 22f** [comment]
> CIO seems to start/stop almost instantly.
Nope. On my different devices with macOS, windows, linux I meet this problems with CIO engine too.
So, I see INFO  `Application - Application started: io.ktor.application.Application` in log, but app start accept requests 4-5 seconds later.
As workaround we can make some healthcheker inside, and after success call to itself write smth at log, but want to understand the nature of this problem.

**Comment by oleg-larshin** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/KTOR-522">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.

**Comment by Stexxe** [comment]
Are able to still reproduce that problem?
